test/Microsoft.TemplateEngine.Cli.UnitTests/TemplateSearchCacheTests.cs

```diff
@@ -51,15 +48,7 @@ public async Task CacheSearchCliSymbolNameFilterTest()
             MockCliNuGetMetadataSearchSource.SetupMockData(mockTemplateDiscoveryMetadata);
             EngineEnvironmentSettings.SettingsLoader.Components.Register(typeof(MockCliNuGetMetadataSearchSource));
 
-            // The template symbol is "Framework" (capital "F"). This checks that the host specific override is applied
-            Dictionary<string, string> rawCommandInputs = new Dictionary<string, string>()
-            {
-                { "framework", "netcoreapp2.0" }
-            };
-            INewCommandInput commandInput = new MockNewCommandInput(rawCommandInputs)
-            {
-                TemplateName = "foo"
-            };
+            INewCommandInput commandInput = new MockNewCommandInput("foo").WithTemplateOption("framework", "netcoreapp2.0");
```

Comment:
minor:
should these values come from a test data set or config? (i.e., `framework` & `netcoreapp2.0`)

Reply:
Test data is coming in the next PR. Many of these changes are done to enable data driven tests